samples/bluetooth/mesh/light/README.rst

```diff
@@ -134,7 +134,7 @@ DFU configuration
 
 .. tabs::
 
-   .. tab:: nRF52840 DK and nRF54L15 PDK
+   .. tab:: nRF52840 DK and nRF54L15 DK
```

Comment:
I see we use nRF54 DKs in several other tab headings. Is there a reason to have nRF54L15 DK here, or can we align with other headers and change it to nRF54 DKs here too?

Reply:
We only have support for nRF54L15 and not nRF54H. And the reason this is different is because the nRF53 has some changes with a app-net core issues. I do not know if that will be the same for nRF54H20. So for now I think it is smart to keep nRF54L15, and when we add support for nRF54H20 we can decide where if that follows nRF53 or these two DK's.